examples/source_separation/README.md

```diff
@@ -8,105 +8,23 @@ This directory contains reference implementations for source separations. For th
 
 ### Overview
 
-To training a model, you can use [`train.py`](./train.py). This script takes the form of
-`train.py [parameters for distributed training] -- [parameters for model/training]`
+To training a model, you can use [`lightning_train.py`](./lightning_train.py). This script takes the form of
+`ligntning_train.py [parameters]`
```

Comment:
```suggestion
`lightning_train.py [parameters]`
```

"h" and "n" do look rather similar...

there are other instances of this below